test/erlcloud_kms_tests.erl

```diff
@@ -173,6 +174,7 @@ output_test(Fun, {Line, {Description, Response, Result}}) ->
      {Line,
       fun() ->
               meck:expect(erlcloud_httpc, request, output_expect(Response)),
+              erlcloud_kms:configure(string:copies("A", 20), string:copies("a", 40)),
```

Comment:
Also, you can call `erlcloud_kms:new()`, but you get less thorough test coverage that way.